All EIPIP Meetings/Meeting 107.md

```diff
@@ -0,0 +1,239 @@
+
+**Sam** [8:48:](https://youtu.be/DgdzaxHWFcA?t=528) No no I'm looking at it but it is not an ERC, it's an interface proposal, we're saying shouldn't be an interface proposal?.
+
+**Pooja** [8:56:](https://youtu.be/DgdzaxHWFcA?t=536) No my bad, so I think I looked on the earlier version where it was mentioned as category ERC now I see an update on category interface, so this can be here because it is a core proposal
+
+**LightClient** [9:10:](https://youtu.be/DgdzaxHWFcA?t=550) I just think it's still an interface and they saw that there was an error and we're like wow, we need to fix that but they didn't read the text that says the ERCs go to a different repo.
+
+**Sam** [9:22:](https://youtu.be/DgdzaxHWFcA?t=562) Yeah this is definitely talking like it has solidity code in it so yeah this is definitely,  I'll close this and redirect them.
+
+**Pooja** [9:33:](https://youtu.be/DgdzaxHWFcA?t=573) Yeah but we might want to consider that because so far we are still having interface category proposal on core, I mean like EIP GitHub repository so it will depend if they want to pursue this as an interface one then it has to.
+
+**Sam** [9:53:](https://youtu.be/DgdzaxHWFcA?t=593) Yeah this is definite like it's a solidity contract this is a clearcut like this is not ERC.
+
+**Poojan** [9:56:](https://youtu.be/DgdzaxHWFcA?t=596) Okay and the number suggest that this is pretty old one and yeah by now it should have received at least, category classification very well we can then perhaps close this one
+
+**Sam** [10:08:](https://youtu.be/DgdzaxHWFcA?t=608) Yeah I'll take care of that.
+
+**Pooja** [10:12:](https://youtu.be/DgdzaxHWFcA?t=612) Perfect uh there are a couple of more, and the first one here is an update to a stagnant proposal 1078 so stagnant definitely can be brought back to life if someone wants to and sees value in that proposal, I am not very sure though, with this pull request they are trying to do that or just fix typo and make a commit in the EIP repository so maybe editor can take a look the number is 540 and this is to fix two proposals 1078 and 1129, though we discussed earlier that we should not welcome any PR to stagnant proposal because um sorry it was for withdrawn yeah it was for withdrawn I guess we should not welcome any changes to withdrawn proposal we can definitely welcome for stagnant, okay any thoughts on merging both of these?
+
+**Sam** [11:44:](https://youtu.be/DgdzaxHWFcA?t=704) I'm just looking through it,  yeah I'm okay with this it's fixing spelling mistakes. I'm just going to first Force merge this like it's not contentious, all right cool.
+
+**Pooja** [12:15:](https://youtu.be/DgdzaxHWFcA?t=735) All right, the next one here seems like spam PR number 8734.
+
+**Sam** [12:31:](https://youtu.be/DgdzaxHWFcA?t=751) Correct, yeah I'll close it .
+
+## 2. Discuss on Moving EIP and ERCs to the right RIP repositories
+
+### They are all moved correctly as required
```

Comment:
Unsure what does it mean here.

Reply:
Please assign header as Agenda items. 

According to the agenda, item number 2 is "2. Other discussions and updates from past meetings". Please verify and add headers as appropriate.